tests/components/rfxtrx/test_sensor.py

```diff
@@ -79,8 +54,8 @@ def test_one_sensor(self):
         )
 
         assert 1 == len(rfxtrx_core.RFX_DEVICES)
-        entity = rfxtrx_core.RFX_DEVICES["sensor_0502"]["Temperature"]
-        assert "Test" == entity.name
+        entity = rfxtrx_core.RFX_DEVICES["sensor_05_02"]["Temperature"]
+        assert "Test Temperature" == entity.name
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)